src/github.org

```diff
@@ -0,0 +1,102 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi, Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu, josh.ting@gatech.edu
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, so you can access old versions at any time without fear
+- Allows multiple developers to work on the same code
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like Dropbox and Google Drive
+
+* What is GitHub?
+- Git \neq GitHub
+- GitHub is a publicly hosted Git server
+- It hosts our projects as "repositories" on the internet
+    - A repository is a group of files that Git collectively tracks
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
+# Emphasize this requirement is for display name on GitHub, not username
+
+[[file:https://i.imgur.com/0cdXQXW.png]]
+
+* Fork and star our repo
+- Navigate to github.com/robojackets/robocup-software
+    - Or the repo of whatever RoboJackets team you are on
+
+[[file:https://i.imgur.com/3CtCTqj.png]]
+
+* What is a fork?
+- Problem: How do I try something out on a project without affecting the project itself?
+- Solution: Forks!
+- A fork is a copy of a repository that exists on your personal account
+- Changes made on a fork will not affect the original repository, and vice versa
+- Your fork will be present on https://github.com/<YOUR-GH-USERNAME>/robocup-software
+
+* Install Git on your computer
+- https://git-scm.com/downloads
+- If on Mac OS, install Homebrew first from https://brew.sh/
+    - Then run `brew install git`
+
+* Git Config
+- These commands set up git to attribute your contributions to our code to you.
+# third command verifies that things are working
+
+  #+BEGIN_SRC shell
+    git config --global user.name "Your Name"
+    git config --global user.email "your@email.here"
+    git config --get --global user.{email,name}
+  #+END_SRC
+
+* Let's grab some code!
+- Go back to the webpage for our GitHub repository
+- Select the clone button, then copy the link displayed
+[[file:https://i.imgur.com/YjWCoqJ.png]]
+
+** Clone the repository
+- In your terminal navigate to your workspace
+
+  #+BEGIN_SRC shell
+  git clone <PASTE-CLIPBOARD-CONTENTS-HERE>
```

Comment:
If we are setting up `software-training` we can make this link explicit. Maybe it's worth it to keep it blank like this though (so we establish they should be copy pasting rather than typing commands blindly). Your call.